AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Subscribe/DataStorePublisher.swift

```diff
@@ -32,3 +35,18 @@ struct DataStorePublisher: DataStoreSubscribeBehavior {
         subject.send(completion: .finished)
     }
 }
+
+protocol ModelSubcriptionBehavior {
+
+    @available(iOS 13.0, *)
+    func publisher(for modelName: ModelName) -> AnyPublisher<MutationEvent, DataStoreError>
+
```

Comment:
As noted above, let's just remove this API and let call sites filter according to their use cases

Reply:
Removed